x/supply/internal/keeper/bank_test.go

```diff
@@ -1,16 +1,18 @@
-package keeper
+package keeper_test
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	. "github.com/cosmos/cosmos-sdk/x/supply/internal/keeper"
```

Comment:
afaiu, dot imports are stylistic choice and strictly unnecessary. Why do we do this?

Reply:
was mostly following what go standard lib did. I can update this

Reply:
it makes sense to use dot notation here as we're on the same folder. I'd keep it

Reply:
I don't understand. This isn't a requirement and imho is a bad practice; i.e. it's not recommended (by the Go team). In the context of DSLs sure, but we're not using a DSL here.

Reply:
updated to using an "keep" alias instead of dot import